src/panels/config/integrations/integration-panels/dynalite/dynalite-config-panel.ts

```diff
@@ -0,0 +1,289 @@
+                <span slot="description">${this.localStr("active_long")}</span>
+                <ha-paper-dropdown-menu
+                  label=${this.localStr("active")}
+                  dynamic-align
+                >
+                  <paper-listbox
+                    slot="dropdown-content"
+                    .selected=${this._activeIndex}
+                    @iron-select=${this._handleActiveSelection}
+                  >
+                    ${this._activeOptions.map(
+                      (option) =>
+                        html`<paper-item .active_config=${option}
+                          >${this.localStr("active_" + option)}</paper-item
+                        >`
+                    )}
+                  </paper-listbox>
+                </ha-paper-dropdown-menu>
+              </ha-settings-row>
+            </div>
+            <div class="card-content">
+              <ha-settings-row .narrow=${this.narrow}>
+                <span slot="heading">${this.localStr("auto_discover")}</span>
+                <span slot="description"
+                  >${this.localStr("auto_discover_long")}</span
+                >
+                <ha-switch
+                  .checked=${this._auto_discover}
+                  @change=${this._handleAutoDiscoverChange}
+                ></ha-switch>
+              </ha-settings-row>
+            </div>
+            <div class="card-content">
+              <ha-settings-row .narrow=${this.narrow}>
+                <span slot="heading">${this.localStr("poll_timer")}</span>
+                <span slot="description"
+                  >${this.localStr("poll_timer_long")}</span
+                >
+                <paper-input
+                  class="flex"
+                  .label=${this.localStr("poll_timer")}
+                  name="poll_timer"
+                  type="number"
+                  .value=${this._poll_timer}
+                  @value-changed=${this._handleChange}
+                ></paper-input>
+              </ha-settings-row>
+            </div>
+            <div class="card-actions">
+              <mwc-button @click=${this._publish}>
+                ${this.localStr("publish")}
+              </mwc-button>
+            </div>
+          </ha-card>
+        </div>
+      </ha-app-layout>
+    `;
+  }
+
+  protected async firstUpdated() {
+    const configEntryId = this._getConfigEntry();
+    if (!configEntryId) return;
+    const response = await this.hass.callWS({
+      type: "dynalite/get_entry",
+      entry_id: configEntryId,
+    });
+    this._entryData = (response as any).data;
+    this._name = this._entryData.name;
+    this._host = this._entryData.host;
+    this._port = this._entryData.port;
+    if (this._entryData.default) {
+      this._fade = this._entryData.default.fade;
+    }
+    const activeMap = {
+      on: 0,
+      true: 0,
+      init: 1,
+      false: 2,
+      off: 2,
+    };
+    this._activeIndex = activeMap[this._entryData.active];
```

Comment:
No need for this, you can just set an value as active instead of the index. Use the `attr-for-selected` attribute of `paper-listbox`

Reply:
can you share with me an example? not a ts expert by any means, so would appreciate to see a place where it is done right.

Reply:
https://github.com/home-assistant/frontend/search?q=attr-for-selected&unscoped_q=attr-for-selected

Reply:
done